### DISCUSSION TITLE
Announcement Disabler

### DISCUSSION THREAD

**Discussion by papercrane1001** [DFHack/dfhack#5017] [Ideas]
Toggleable plugin that allows you to disable certain announcements -- thinking of you, "X was stung by a bee!" and "Y has made a masterwork trade good!"

**Comment by ab9rf** [comment]
This is already largely offered by the existing "Announcements" configuration in the vanilla settings screen. There is already a vanilla toggle for making masterwork goods, and bee stings are a specific type of "vermin bite" which can also be toggled on or off in the vanilla announcement configuration.

Providing a greater degree of granularity than what the vanilla configuration allows is _difficult_.

**Reply by papercrane1001** [reply]
I had not noticed that tab at all.  Thank you, and I will be more careful about any further ideas.  

The only way I can see to extend this concept would be to add a button on the actual announcements that would allow you to disable similar announcements, useful if you're getting spammed with something, but I don't see that as nearly as much of a priority.
